apps/climatemappedafrica/src/components/DropdownSearch/DownloadSearch.js

```diff
@@ -175,6 +175,7 @@ function DropdownSearch({
             {suggestions.map(({ name, code }) => (
               <ListItem
                 component={Link}
+                href={`${hrefProp}/${code}`}
```

Comment:
If we're using `href=${hrefProp}/${code` then we don't need `onClick` anymore because the page will navigate and we already have server-side rendering.

If we're not doing the necessary clean-up in this PR, lets and an issue to track it